core/src/androidMain/kotlin/Scanner.kt

```diff
@@ -13,13 +13,15 @@ public class ScanFailedException internal constructor(
     public val errorCode: Int
 ) : IllegalStateException("Bluetooth scan failed with error code $errorCode")
 
+public actual fun Scanner(): Scanner = AndroidScanner()
+
 public class AndroidScanner internal constructor() : Scanner {
 
     private val bluetoothAdapter = BluetoothAdapter.getDefaultAdapter()
         ?: error("Bluetooth not supported")
 
-    public override val peripherals: Flow<Advertisement> = callbackFlow {
-        check(bluetoothAdapter.isEnabled) { "Bluetooth is disabled" } // todo: Exception type.
+    public override val advertisements: Flow<Advertisement> = callbackFlow {
+        check(bluetoothAdapter.isEnabled) { "Bluetooth is disabled" }
```

Comment:
Is the todo no longer necessary?

Reply:
Ya, I thought about it, and `IllegalStateException` seems appropriate here.
We can change it later if we find a compelling reason, but I don't think it warrants a `todo`.